omod/src/main/webapp/pages/metadata/privileges/privilege.gsp

```diff
@@ -55,35 +55,42 @@
     });
 </script>
 
-<h1>
-    <h3>${ ui.message((createPrivilege) ? "adminui.addNewPrivilege.label" : "adminui.editPrivilege.label")}</h3>
-</h1>
+<% if(privilege == null) {%>
+    <div align="center">
+        <h1>${ui.message("No privilege found with name ")}"${unknownPrivilegeName}"</h1>
```

Comment:
thanks @jnsereko  no need of adding this ui.message("No privilege found with name ")}"

Reply:
Thanks @HerbertYiga for the review..
Man this is far a better alternative.